sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DocumentCollection.java

```diff
@@ -487,7 +487,7 @@ public String toJson() {
 
     public static class SerializableDocumentCollection implements SerializableWrapper<DocumentCollection> {
         private static final long serialVersionUID = 2l;
-        private static final ObjectMapper OBJECT_MAPPER = Utils.getSimpleObjectMapper();
+        private static final ObjectMapper OBJECT_MAPPER = Utils.getSimpleObjectMapperWithAllowDuplicates();
```

Comment:
this default behavior change might be breaking for some scenarios?

Reply:
No when? We only use this ObjectMapper instance ever to serialize/deserialize container metadata